cluster/addons/storage-class/caph/default.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: ceph
+provisioner: kubernetes.io/rbd
+# parameters:
```

Comment:
Do we really need these comments?

Reply:
@dixux What advice can be given to me to make it better.